src/lib/models/user.model.ts

```diff
@@ -6,5 +6,6 @@ export type User = {
   createdAt: Date;
   updatedAt: Date;
   email: string;
+  notifications: string[];
```

Comment:
Something to think about: Could the notification experience be better if we store more useful information, such as a link to the adventure?